### ISSUE TITLE
some entries have leading "=" in header in navigator 

### ISSUE THREAD

**Issue by paregorios** [DCLP/dclpxsltbox#316]
E.g.: http://litpap.info/dclp/59495 (as http://dclp.github.io/dclpxsltbox/output/dclp/60/59495.html). 

What's missing?

**Comment by paregorios** [comment]
full list of victims:

```
./output/dclp/110/109360.html
./output/dclp/110/109375.html
./output/dclp/113/112705.html
./output/dclp/118/117905.html
./output/dclp/118/117906.html
./output/dclp/118/117907.html
./output/dclp/118/117908.html
./output/dclp/118/117909.html
./output/dclp/120/119281.html
./output/dclp/129/128541.html
./output/dclp/129/128916.html
./output/dclp/131/130194.html
./output/dclp/141/140271.html
./output/dclp/141/140289.html
./output/dclp/141/140296.html
./output/dclp/141/140302.html
./output/dclp/141/140303.html
./output/dclp/141/140551.html
./output/dclp/221/220437.html
./output/dclp/221/220475.html
./output/dclp/221/220476.html
./output/dclp/221/220477.html
./output/dclp/221/220478.html
./output/dclp/221/220479.html
./output/dclp/221/220480.html
./output/dclp/244/243957.html
./output/dclp/244/243958.html
./output/dclp/244/243959.html
./output/dclp/244/243960.html
./output/dclp/244/243961.html
./output/dclp/244/243962.html
./output/dclp/244/243963.html
./output/dclp/244/243964.html
./output/dclp/244/243965.html
./output/dclp/245/244104.html
./output/dclp/245/244105.html
./output/dclp/245/244106.html
./output/dclp/245/244107.html
./output/dclp/245/244108.html
./output/dclp/60/59112.html
./output/dclp/60/59360.html
./output/dclp/60/59434.html
./output/dclp/60/59435.html
./output/dclp/60/59436.html
./output/dclp/60/59446.html
./output/dclp/60/59447.html
./output/dclp/60/59450.html
./output/dclp/60/59452.html
./output/dclp/60/59467.html
./output/dclp/60/59468.html
./output/dclp/60/59490.html
./output/dclp/60/59491.html
./output/dclp/60/59492.html
./output/dclp/60/59493.html
./output/dclp/60/59494.html
./output/dclp/60/59495.html
./output/dclp/60/59496.html
./output/dclp/60/59497.html
./output/dclp/60/59498.html
./output/dclp/60/59499.html
./output/dclp/60/59500.html
./output/dclp/60/59501.html
./output/dclp/60/59502.html
./output/dclp/60/59503.html
./output/dclp/60/59504.html
./output/dclp/60/59505.html
./output/dclp/60/59506.html
./output/dclp/60/59721.html
./output/dclp/60/59737.html
./output/dclp/60/59738.html
./output/dclp/60/59739.html
./output/dclp/60/59740.html
./output/dclp/60/59741.html
./output/dclp/60/59742.html
./output/dclp/60/59743.html
./output/dclp/60/59744.html
./output/dclp/60/59746.html
./output/dclp/60/59747.html
./output/dclp/60/59748.html
./output/dclp/60/59749.html
./output/dclp/60/59751.html
./output/dclp/60/59752.html
./output/dclp/60/59753.html
./output/dclp/60/59754.html
./output/dclp/60/59756.html
./output/dclp/60/59758.html
./output/dclp/60/59759.html
./output/dclp/60/59760.html
./output/dclp/60/59761.html
./output/dclp/61/60497.html
./output/dclp/62/61438.html
./output/dclp/62/61600.html
./output/dclp/63/62380.html
./output/dclp/63/62381.html
./output/dclp/63/62382.html
./output/dclp/63/62383.html
./output/dclp/63/62384.html
./output/dclp/63/62385.html
./output/dclp/63/62386.html
./output/dclp/63/62387.html
./output/dclp/63/62388.html
./output/dclp/63/62389.html
./output/dclp/63/62390.html
./output/dclp/63/62391.html
./output/dclp/63/62393.html
./output/dclp/63/62394.html
./output/dclp/63/62400.html
./output/dclp/63/62402.html
./output/dclp/63/62403.html
./output/dclp/63/62404.html
./output/dclp/63/62405.html
./output/dclp/63/62406.html
./output/dclp/63/62407.html
./output/dclp/63/62408.html
./output/dclp/63/62409.html
./output/dclp/63/62411.html
./output/dclp/63/62414.html
./output/dclp/63/62416.html
./output/dclp/63/62419.html
./output/dclp/63/62422.html
./output/dclp/63/62423.html
./output/dclp/63/62424.html
./output/dclp/63/62425.html
./output/dclp/63/62426.html
./output/dclp/63/62427.html
./output/dclp/63/62428.html
./output/dclp/63/62429.html
./output/dclp/63/62430.html
./output/dclp/63/62431.html
./output/dclp/63/62433.html
./output/dclp/63/62434.html
./output/dclp/63/62435.html
./output/dclp/63/62436.html
./output/dclp/63/62437.html
./output/dclp/63/62439.html
./output/dclp/63/62440.html
./output/dclp/63/62441.html
./output/dclp/63/62442.html
./output/dclp/63/62443.html
./output/dclp/63/62444.html
./output/dclp/63/62445.html
./output/dclp/63/62447.html
./output/dclp/63/62448.html
./output/dclp/63/62450.html
./output/dclp/63/62457.html
./output/dclp/63/62459.html
./output/dclp/63/62460.html
./output/dclp/63/62462.html
./output/dclp/63/62463.html
./output/dclp/63/62464.html
./output/dclp/63/62465.html
./output/dclp/63/62466.html
./output/dclp/63/62467.html
./output/dclp/63/62468.html
./output/dclp/63/62469.html
./output/dclp/63/62471.html
./output/dclp/63/62474.html
./output/dclp/63/62475.html
./output/dclp/63/62476.html
./output/dclp/63/62477.html
./output/dclp/63/62478.html
./output/dclp/63/62479.html
./output/dclp/63/62480.html
./output/dclp/63/62481.html
./output/dclp/63/62483.html
./output/dclp/63/62485.html
./output/dclp/63/62486.html
./output/dclp/63/62487.html
./output/dclp/63/62488.html
./output/dclp/63/62489.html
./output/dclp/63/62492.html
./output/dclp/63/62493.html
./output/dclp/63/62494.html
./output/dclp/63/62495.html
./output/dclp/63/62498.html
./output/dclp/63/62499.html
./output/dclp/63/62500.html
./output/dclp/63/62503.html
./output/dclp/63/62505.html
./output/dclp/63/62508.html
./output/dclp/63/62509.html
./output/dclp/63/62510.html
./output/dclp/63/62511.html
./output/dclp/63/62512.html
./output/dclp/63/62513.html
./output/dclp/63/62514.html
./output/dclp/63/62516.html
./output/dclp/63/62661.html
./output/dclp/63/62682.html
./output/dclp/64/63123.html
./output/dclp/64/63132.html
./output/dclp/64/63133.html
./output/dclp/64/63134.html
./output/dclp/64/63137.html
./output/dclp/64/63138.html
./output/dclp/64/63139.html
./output/dclp/64/63140.html
./output/dclp/64/63141.html
./output/dclp/64/63142.html
./output/dclp/64/63143.html
./output/dclp/64/63144.html
./output/dclp/64/63146.html
./output/dclp/64/63147.html
./output/dclp/64/63148.html
./output/dclp/64/63149.html
./output/dclp/64/63151.html
./output/dclp/64/63152.html
./output/dclp/64/63153.html
./output/dclp/64/63155.html
./output/dclp/64/63156.html
./output/dclp/64/63157.html
./output/dclp/64/63158.html
./output/dclp/64/63160.html
./output/dclp/64/63161.html
./output/dclp/64/63162.html
./output/dclp/64/63163.html
./output/dclp/64/63164.html
./output/dclp/66/65542.html
./output/dclp/66/65543.html
./output/dclp/66/65659.html
./output/dclp/67/66487.html
./output/dclp/67/66488.html
./output/dclp/70/69067.html
./output/dclp/70/69730.html
./output/dclp/70/69731.html
./output/dclp/70/69732.html
./output/dclp/81/80754.html
./output/dclp/81/80755.html
./output/dclp/81/80756.html
./output/dclp/81/80758.html
./output/dclp/81/80759.html
./output/dclp/91/90886.html
```

**Comment by paregorios** [comment]
https://github.com/DCLP/navigator/commit/f1e6564c53a119621353f3dc3da1ccb21ae5eba1

**Comment by hcayless** [comment]
I've got a question about this one: (happened to come across my screen while waiting for a reindex) is there a reason why you wouldn't want all of the principal editions listed in the header? E.g. for http://litpap.info/dclp/38657 is 

### sb.16.12403 = HGV SB 16 12403 = Trismegistos 38657

better than 

### SB 16 12403 = P. Rainer Unterricht (MPER N.S. 15) 171 = Trismegistos 38657 = LDAB 6747

which can be achieved simply by removing the positional predicates at line [706](https://github.com/DCLP/navigator/blob/57d6df5050306d0b8fe70612fe279dff5a15d3d7/pn-xslt/MakeHTML.xsl#L706) and moving the `<xsl:text> = </xsl:text>` inside the for-each? Incidentally, the latter gets rid of the leading '=' too.

**Comment by rogerbagnall** [comment]
That's a much more elegant solution. My only uncertainty would be whether including the LDAB number is useful.

**Comment by hcayless** [comment]
@rogerbagnall the LDAB number is a link to the LDAB database in TM (the link didn't come across in my copy/paste). But that makes me realize the LDAB link does not get generated if the DCLP record is attached to a DDb text, as this example is. So that's probably another bug...

**Comment by paregorios** [comment]
This further complicates a number of rather complicated (and previously contradictory) requirements that have been enunciated piecemeal across a large number of tickets (e.g., #31, which @wsalesky and I have been working on for days).

**Comment by paregorios** [comment]
Please open new tickets for new requirements. This ticket is only about leading "=" in header, which is fixed with https://github.com/DCLP/navigator/commit/f1e6564c53a119621353f3dc3da1ccb21ae5eba1, which is currently on [the issue31 branch of DCLP navigator](https://github.com/DCLP/navigator/tree/issue31) and will be merged once #31 is resolved.
